packages/block-library/src/button/edit.native.js

```diff
@@ -173,23 +173,33 @@ class ButtonEdit extends Component {
 
 	getBackgroundColor() {
 		const { backgroundColor, attributes } = this.props;
-		const { style } = attributes;
-
+		const { style, gradient, customGradient } = attributes;
+		const fallbackBackgroundColor = styles.defaultButton.backgroundColor;
```

Comment:
Same as for `fallbackTextColor` i think we don't need a const for that :)

Reply:
Ok